src/Microsoft.Bcl.AsyncInterfaces/tests/Configurations.props

```diff
@@ -1,7 +1,9 @@
 ﻿<Project DefaultTargets="Build">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
+      <!-- netstandard; -->
+      netcoreapp;
+      uap;
```

Comment:
This kind of defeats the purpose of these tests, right? :)

Should we instead just exclude ConfiguredCancelableAsyncEnumerableTests.netcoreapp.cs from being included until netstandard2.1 is updated?

Reply:
It does defeat the purpose, but it is a temporary change, so I'm fine with either removing the tests or leaving this as is and fixing once netstandard contains the type. As you prefer.

Reply:
Thanks.  I prefer the excluding-these-few-tests approach.